### ISSUE TITLE
Rename libraries to avoid clashes between sudirectories in Subprocesses

### ISSUE THREAD

**Issue by valassi** [madgraph5/madgraph4gpu#397]
This is a good suggestion by @roiser: processes like pptt have two subdirectories ggtt and uutt, and both produce libraries with the same name in the same location. 

One option is to append the process name to each library. 

For instance “libmg5amc_cxx_p1_sigma_sm_gg_ttx.so”. This is easy to implement also in cmake, but may be a bit long.

Another alternative, shorter but maybe less easy in cmake,
```
> libmg5amc_gg_ttx_common.so
> libmg5amc_gg_ttx_cu.so
> libmg5amc_gg_ttx_cxx.so
> libmg5amc_uux_ttx_common.so
> libmg5amc_uux_ttx_cu.so
> libmg5amc_uux_ttx_cxx.so
```

To be investigated

**Comment by valassi** [comment]
I have made a simple patch in the Makefile to have the following
```
[avalassi@itscrd70 gcc10.2/cvmfs] /data/avalassi/GPU2020/madgraph4gpuX/epochX/cudacpp/pp_tt.auto> ls -1 lib/
./
../
.build.512y_d_inl0_hrd0_hasCurand
libmg5amc_common.so*
libmg5amc_gg_ttx_cu.so*
libmg5amc_gg_ttx_cxx.so*
libmg5amc_uux_ttx_cu.so*
libmg5amc_uux_ttx_cxx.so*
```

**Comment by valassi** [comment]
This is fixed in #396 that I am about to merge. Closing.

In case we can decide to change the names if cmake requires a different naming strategy
